sdk/metricsadvisor/azure-ai-metricsadvisor/azure/ai/metricsadvisor/models/_models.py

```diff
@@ -105,6 +105,58 @@
     )
     from .._metrics_advisor_administration_client import DataFeedSourceUnion
 
+class DictMixin:
```

Comment:
Should we move it into core?

Reply:
I think i'm actually going to close this PR. I don't think we need to though, the DPG models we're workign on will have this built in, and they're going to be in core